libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -151,8 +153,18 @@ static uint8_t eeprom_buffer[E2END + 1] __attribute__((aligned(8))) = {0};
   */
 uint8_t eeprom_read_byte(const uint32_t pos)
 {
+#if defined(DATA_EEPROM_BASE)
+  __IO uint8_t data = 0;
+  if (pos <= (DATA_EEPROM_END - DATA_EEPROM_BASE)) {
+    /* with actual EEPROM, pos is a relative address */
+    data = *(__IO uint8_t *)(DATA_EEPROM_BASE + pos);
+
+    return (uint8_t)data;
+  }
```

Comment:
warning: control reaches end of non-void function [-Wreturn-type]

```suggestion
  }
  return (uint8_t)data;
```

Reply:
changed